docs/setup/pick-right-solution.md

```diff
@@ -130,7 +130,7 @@ These solutions are combinations of cloud providers and operating systems not co
 * [Fedora (Single Node)](/docs/getting-started-guides/fedora/fedora_manual_config/)
 * [Fedora (Multi Node)](/docs/getting-started-guides/fedora/flannel_multi_node_cluster/)
 * [Kubernetes on Ubuntu](/docs/getting-started-guides/ubuntu/)
-* [CoreOS on AWS or GCE](/docs/getting-started-guides/coreos/)
+* [CoreOS on Bare Metal](/docs/getting-started-guides/coreos/)
```

Comment:
I see why you make the change here, and the original is sort of wrong, too -- but the thing is, the links all point to the same topic/file, which is the one you've correctly changed the title on. I'd recommend

s/CoreOS on Bare Metal/CoreOS

because /docs/getting-started-guides/coreos points to a topic/file that's more generic than just bare metal. Make sense?